src/components/publiq-ui/Alert.js

```diff
@@ -32,15 +21,22 @@ const Alert = ({
   ...props
 }) => {
   return (
-    <StyledBootstrapAlert
+    <BootstrapAlert
+      forwardedAs={Box}
```

Comment:
Here we do `<BootstrapAlert forwardedAs={Box} ...>` and in another component we do `<Box forwardedAs={BootstrapBadge}>`. In the other components we also switch between both approaches. What is the reasoning behind this? When should we use which approach? @geoffreydhuyvetters can you clarify this maybe? Simon doesn't know for sure he told me.